OpenEdXMobile/res/layout/empty_screen_layout.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
as per the Figam designs screen should show the round corner card. instated of full screen.